pkg/minikube/command/exec_runner.go

```diff
@@ -39,7 +40,7 @@ type ExecRunner struct{}
 // RunCmd implements the Command Runner interface to run a exec.Cmd object
 func (*ExecRunner) RunCmd(cmd *exec.Cmd) (*RunResult, error) {
 	rr := &RunResult{Args: cmd.Args}
-	glog.Infof("(ExecRunner) Run:  %v", rr.Command())
```

Comment:
I personally like that (ExecRunner) prrfix. In cases which a driver might use two runners it helps to know what is run in where... Less confusing.

Reply:
The log line already mentions exec_runner.go.